RELEASE-NOTES.md

```diff
@@ -114,7 +114,8 @@ This includes API changes we did not warn about since at least `3.11.0` (2021-01
   - Modify how particle weights are computed. This improves accuracy of the modeled function (see [5177](https://github.com/pymc-devs/pymc3/pull/5177)).
   - Improve sampling, increase default number of particles [5229](https://github.com/pymc-devs/pymc3/pull/5229).
 - `pm.Data` now passes additional kwargs to `aesara.shared`. [#5098](https://github.com/pymc-devs/pymc/pull/5098)
-- ...
+- The new `pm.find_constrained_prior` function can be used to find optimized prior parameters of a distribution under some
```

Comment:
Still says "optimized"

Reply:
Well here I think it's appropriate: the function really returns "optimized prior parameters under some constraints" -- it doesn't say that they are perfect, or the best, or optimal